plugin/trino-influxdb/src/main/java/io/trino/plugin/influxdb/InfluxSplit.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.HostAddress;
+import io.trino.spi.connector.ConnectorSplit;
+import org.openjdk.jol.info.ClassLayout;
+
+import java.util.List;
+
+import static io.airlift.slice.SizeOf.estimatedSizeOf;
+import static java.util.Objects.requireNonNull;
+
+public class InfluxSplit
+        implements ConnectorSplit
+{
+    private static final long INSTANCE_SIZE = ClassLayout.parseClass(InfluxSplit.class).instanceSize();
+
+    private final String endpoint;
+    private final List<HostAddress> addresses;
+
+    @JsonCreator
+    public InfluxSplit(@JsonProperty("endpoint") String endpoint)
+    {
+        this.endpoint = requireNonNull(endpoint, "endpoint is null");
```

Comment:
We don't have to pass `endpoint` here

Reply:
Ok, I copy that way from mongoSplit,  but if we remove the endpoint , how to get connect to influx server.

Reply:
Get it . I put the endpoint to List<Address> for method getAddress here. I see ConnectorSplit says "may be used as hints by the scheduler during splits assignment when isRemotelyAccessible is true".
So can i make getAddress return an empty list? or should throw unsupported exception?